HW/Oleksandr Liashenko/HW09_Task1.py

```diff
@@ -0,0 +1,20 @@
+from random import randint
+
+rand_num = randint(1, 100)
+tries = 10
+
+while tries > 0:
+    usr_num = int(input(f"Enter a number ({tries} left): "))
+    if usr_num == rand_num:
+        print("Congratulations!")
+        break
+    elif tries == 1:
+        print("Game over")
+        break
```

Comment:
Dont understand this block, anyway while will be stoped

Reply:
This is a weird part, but it prevents printing "Try a greater number" or "Try a lesser number" in the case when this try is the last one (if I move this block outside the 'while' loop)